The odbcinst.exp file shipped with 2.2.11 is a few symbols shy of a load.
We could try adding symbols to it, but the safest and most backward-compatible
solution is to not use it.  Hopefully the community will have this sorted
properly by 2.2.12, and we can get rid of this kluge.


diff -Naur unixODBC-2.2.11.orig/odbcinst/Makefile.am unixODBC-2.2.11/odbcinst/Makefile.am
--- unixODBC-2.2.11.orig/odbcinst/Makefile.am	2005-02-07 05:14:01.000000000 -0500
+++ unixODBC-2.2.11/odbcinst/Makefile.am	2005-04-08 13:48:45.719529522 -0400
@@ -11,8 +11,7 @@
 
 libodbcinst_la_LDFLAGS = \
     -version-info @LIB_VERSION@ \
-    -export-dynamic \
-    -export-symbols @srcdir@/odbcinst.exp
+    -export-dynamic
 
 libodbcinst_la_LIBADD = \
 	../ini/libinilc.la \

